releasenotes/notes/dagcircuit-separable-circuits-142853e69f530a16.yaml

```
@@ -0,0 +1,15 @@
+---
+features:
+  - |
+    Introduced a new :class:`.DAGCircuit` class method,
+    :meth:`~.separable_circuits`, which returns a list of :class:`.DAGCircuit` objects,
+    one for each set of connected qubits which have no gates connecting them to another
+    set.
```

```suggestion
    Introduced a new :class:`.DAGCircuit` method,
    :meth:`~.DAGCircuit.separable_circuits`, which returns a list of :class:`.DAGCircuit` objects,
    one for each set of connected qubits which have no gates connecting them to another
    set.
```

(first change because "class method" means something particular, and second just makes the lookup safer in case any other class in the future happens to add a `separable_circuits` method.)